docs/source/telegram.inline-tree.rst

```diff
@@ -1,6 +1,12 @@
 Inline Mode
 -----------
 
+The following methods and objects allow your bot to work in `inline mode`_.
+Please see our `Introduction to Inline bots <https://core.telegram.org/bots/inline>`_ for more details.
```

Comment:
```suggestion
Please see Telegrams `Introduction to Inline bots <https://core.telegram.org/bots/inline>`_ for more details.
```
kindly double check your changes for this kind of reference. We should make sure that with "we" we always mean PTB.